### PR TITLE
Ticket #220 Update search box functionality to be consistent with Drug Dictionary

### LINKED ISSUE #220: Update search box functionality to be consistent with Drug Dictionary
### Issue description
- The Search button is not full width like it is on drug dictionary.
- Autosuggest dropdown should be dropped either at the button or above search box based on screen
-  Remove auto highlight of the first autosuggest item - to enable users to hit enter and search without selecting an item

> **ESTIMATE** TBD

### What's the expected change?
-The search button should be full width on mobile (<641).
- no autohighlight of first item
- dropdown changes its position based on a screen

### What's the current functionality?
-The search button is not full width on mobile.

### What's the updated acceptance criteria?
```gherkin
```

### Additional details / screenshot
https://react-app-dev.cancer.gov/drug-dictionary-app/pr-81/def/bevacizumab 

### Related Tickets
* https://github.com/NCIOCPL/drug-dictionary-app/issues/57

### PR DESCRIPTION
**Pull request by alerosad**
Closes #220 
Update search box functionality to be consistent with Drug Dictionary

- edited search box width along with text area width, when the screen
width is < 614

- adjusted dropdown menu so that it goes up when screen width is <614

- removed autohighlight feature

- adjusted tests accordingly

### CODE REVIEW

**Comment by alerosad** [comment]
https://react-app-dev.cancer.gov/glossary-app/pr-230/

File: src/components/atomic/TextInput/TextInput.scss [hunk 1]
```diff
@@ -13,7 +13,7 @@
 	&[type='url'],
 	&[type='week'] {
 		background-color: white;
-		margin: 0;
+		margin-bottom: 20	px;
```
Comment (arcepaul) [inline src/components/atomic/TextInput/TextInput.scss:16]: weird spacing going on here

File: src/components/molecules/search/Search.scss [hunk 2]
```diff
@@ -14,6 +14,12 @@
 	.dictionary-search-input {
 		display: inline-block;
 		width: 65%;
+
+		@media screen and (max-width: 614px) {
+
```
Comment (arcepaul) [inline src/components/molecules/search/Search.scss:19]: Tighten this up and/or run formatter. (option + shift + F)

File: src/components/molecules/search/__tests__/Search.test.js [hunk 3]
```diff
@@ -230,7 +230,8 @@ describe('<Search /> English', () => {
 				fireEvent.click(searchButton);
 			});
 			const expectedLocationObject = {
-				pathname: `/search/meta-analysis/`,
```
Comment (arcepaul) [inline src/components/molecules/search/__tests__/Search.test.js:233]: did this linkage change?

File: src/components/atomic/TextInput/TextInput.scss (hunk 1 above)
Comment (alerosad) [reply src/components/atomic/TextInput/TextInput.scss:16]: I'll go ahead and fix that, not sure why there's so much spacing

File: src/components/molecules/search/Search.scss (hunk 2 above)
Comment (alerosad) [reply src/components/molecules/search/Search.scss:19]: Ok

File: src/components/molecules/search/__tests__/Search.test.js (hunk 3 above)
Comment (alerosad) [reply src/components/molecules/search/__tests__/Search.test.js]: Whoops I changed it a while ago to test something and forgot to switch it back. Fixed now.

Comment (alerosad) [reply src/components/molecules/search/__tests__/Search.test.js]: Ah wait getting an issue on one of the unit tests that I missed.

Comment (alerosad) [reply src/components/molecules/search/__tests__/Search.test.js]: Ok should be fine now

**Comment by kate-mashkina** [comment]
There are two things QA noticed :
Search button text is smaller than drug dict
search button should be full widths starting at 640 and below (currently its at <=614)

**Comment by blilianyu** [comment]
Can you reduce the space between the search input field and the button? There is too much space. 

Screen Shot 2021-01-25 at 9 41 05 AM

**Comment by alerosad** [comment]
> There are two things QA noticed :
> Search button text is smaller than drug dict
> search button should be full widths starting at 640 and below (currently its at <=614)



> Can you reduce the space between the search input field and the button? There is too much space on screen size 614 and less.
> 
> Screen Shot 2021-01-25 at 9 41 05 AM

Sure will do

**Comment by alerosad** [comment]
> There are two things QA noticed :
> Search button text is smaller than drug dict
> search button should be full widths starting at 640 and below (currently its at <=614)

Ok, will do

**Comment by alerosad** [comment]
https://react-app-dev.cancer.gov/glossary-app/pr-230 @blilianyu @mworrest

**Comment by alinai** [comment]
The drop-down menu is blocking the Search box. @blilianyu, can you please check?

image

image

image

**Comment by arcepaul** [comment]
@alinai @mworrest @blilianyu @VictoriaSunNIH 
Latest with linting changes is now available on react app dev (https://react-app-dev.cancer.gov/glossary-app/pr-230/)

**Comment by VictoriaSunNIH** [comment]
@arcepaul , @alerosad - I'm still seeing issues with the dropdown on mobile not going in the right direction - "dropdown changes its position based on a screen" is still not happening for me.

**Comment by alerosad** [comment]
I made that adjustment that @alinai mentioned

**Comment by alerosad** [comment]
Wait, forgot to add in newest changes from develop, will fix

**Comment by alerosad** [comment]
When I pull in the most recent commit from develop, I can't seem to run the unit tests without receiving:
image

**Comment by alerosad** [comment]
https://react-app-dev.cancer.gov/glossary-app/pr-230 @mworrest @blilianyu @VictoriaSunNIH

**Comment by VictoriaSunNIH** [comment]
@KateMashkinaNIH - reminder to please QA this one so we can call it done if it passes :)

**Review by KateMashkinaNIH — APPROVED** [review]

**Pull request merged**